pkgs/by-name/mb/mbrola/package.nix

```diff
@@ -1,52 +1,47 @@
-{ stdenv, lib, fetchFromGitHub, runCommandLocal, mbrola-voices }:
-
-let
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  mbrola-voices,
+  runCommandLocal,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
   pname = "mbrola";
   version = "3.3";
 
-  meta = with lib; {
-    license = licenses.agpl3Plus;
-    maintainers = with maintainers; [ davidak ];
-    platforms = platforms.all;
-    description = "Speech synthesizer based on the concatenation of diphones";
-    homepage = "https://github.com/numediart/MBROLA";
+  src = fetchFromGitHub {
+    owner = "numediart";
+    repo = "MBROLA";
+    rev = finalAttrs.version;
+    hash = "sha256-/iOKBnQM17RwFNt/CBZ6gPduNvqJ2DAOyIsJ/c1+BvE=";
   };
 
-  bin = stdenv.mkDerivation {
-    pname = "${pname}-bin";
-    inherit version;
-
-    src = fetchFromGitHub {
-      owner = "numediart";
-      repo = "MBROLA";
-      rev = version;
-      sha256 = "1w86gv6zs2cbr0731n49z8v6xxw0g8b0hzyv2iqb9mqcfh38l8zy";
-    };
-
+  makeFlags = [
     # required for cross compilation
-    makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ];
-
-    installPhase = ''
-      runHook preInstall
-      install -D Bin/mbrola $out/bin/mbrola
-      rm -rf $out/share/mbrola/voices/*
-      runHook postInstall
+    "CC=${stdenv.cc.targetPrefix}cc"
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    install -D Bin/mbrola $out/bin/mbrola
+    runHook postInstall
+  '';
+
+  passthru = {
+    combined = runCommandLocal "mbrola-combined-${finalAttrs.version}" { inherit (finalAttrs) meta; } ''
```

Comment:
We could make a nicer git history if we didn't include this intermediate step in the commits.

Reply:
If we do not include this step then some of the commits will be broken.